src/modules/search/components/result/SearchResults.tsx

```diff
@@ -3,20 +3,35 @@ import css from './SearchResults.module.scss';
 import ProjectCard from './ProjectCard';
 import { Project } from '../../../../types/common';
 import { useTranslation } from 'react-i18next';
+import { Button, IconMap } from 'hds-react';
 
 type Props = {
   searchResults: Project[];
+  openMap: () => void;
 };
 
-const SearchResults = ({ searchResults }: Props) => {
+const SearchResults = ({ searchResults, openMap }: Props) => {
   const { t } = useTranslation();
 
   return (
     <div className={css.container}>
-      <div className={css.titleContainer}>
-        <h1>{t('SEARCH:free-apartments')}</h1>
-        <div className={css.resultsCount}>
-          {t('SEARCH:total')} {searchResults.length} {t('SEARCH:apartments')}
+      <div style={{ display: 'flex', justifyContent: 'space-between', flexWrap: 'wrap', padding: '62px 48px 32px' }}>
```

Comment:
Is there some magic going on here I don't see or why is the styles inline here...?